app/models/queen.rb

```diff
@@ -1,9 +1,15 @@
 class Queen < Piece
 
+  def valid_move?(x, y)
+    unless super(x, y)
```

Comment:
if you use `super` without parens, it will pass up the arguments the original method was called with.

in this case it's okay to just do `super`, but if you wanna be super (pun intended) clear, go ahead and leave it as is.